pkgs/tools/X11/xssstate/default.nix

```
@@ -4,29 +4,33 @@
 , libX11
 , libXScrnSaver
 }:
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "xssstate";
   #
   # Use the date of the last commit, since there were bug fixes after the 1.1
   # release.
   #
-  version = "unstable-2022-09-24";
+  version = "1.1-unstable-2022-09-24";
```

```suggestion
  version = "unstable-2022-09-24";
```
If you need 1.1 in the version information, please add it to installFlags.

---

No, `1.1-unstable-2022-09-24` is perfectly valid.
See [`pkgs/README.md#package-naming`](https://github.com/NixOS/nixpkgs/blob/cd3c88ad04b4fc98112dee77aa6eb356aa8419a5/pkgs/README.md#package-naming) @natsukium^

---

Wow, it seems to have been updated before I knew it.
Thanks for sharing.